ush/python/pygfs/task/nexus_emission.py

```diff
@@ -0,0 +1,486 @@
+#!/usr/bin/env python3
+
+import os
+import re
+from collections import defaultdict
+import xarray as xr
+import subprocess
+import numpy as np
+import cftime
+from logging import getLogger
+from typing import Dict, Any, Union, List
+from dateutil.rrule import DAILY, HOURLY, rrule
+from jinja2 import Environment, FileSystemLoader
+from pprint import pprint 
+from wxflow import (AttrDict,
+                    FileHandler,
+                    parse_j2yaml,
+                    logit,
+                    Task,
+                    to_timedelta,
+                    WorkflowException,
+                    Executable, which)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class NEXUSEmissions(Task):
+    """NEXUS Emissions pre-processing Task
+    """
+
+    @logit(logger, name="NEXUSEmissions")
+    def __init__(self, config: Dict[str, Any]) -> None:
+        """Constructor for the NEXUS Emissions task
+
+        Parameters
+        ----------
+        config : Dict[str, Any]
+            Incoming configuration for the task from the environment
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(config)
+
+        self.task_config = AttrDict(config)
+        pprint(self.task_config)
+
+        self.AERO_INPUTS_DIR = self.task_config.get('AERO_INPUTS_DIR', None)
+        self.COMOUT_CHEM_INPUT = self.task_config.get('COMOUT_CHEM_INPUT', None)
+
+        # get the nforecast hours - gcdas will use FHMAX and gcafs will use FHMAX_GFS
+        if 'das' in self.task_config['RUN']:
+            nforecast_hours = self.task_config["FHMAX"]
+        else:
+            nforecast_hours = self.task_config["FHMAX_GFS"]
+
+        # Create start date based on SDATE
+        self.start_date = self.task_config["CDATE"]
+        self.total_hrs = nforecast_hours + 3
```

Comment:
why +3? can you add a comment to explain this line

Reply:
Really it could be +1 but I did +3 just to be safe